public/app/features/scopes/selector/ScopesSelectorService.ts

```diff
@@ -214,8 +214,8 @@ export class ScopesSelectorService extends ScopesServiceBase<ScopesSelectorServi
     return this.collapseNode(scopeNodeId);
   };
 
-  changeScopes = (scopeNames: string[]) => {
-    return this.applyScopes(scopeNames.map((id) => ({ scopeId: id })));
+  changeScopes = (scopeNames: string[], parentNodeId?: string) => {
```

Comment:
Not a fan of changing the public API for this - Will this also be usable for the other cases, such as choosing which parent node title to display next to the selector?